stable/metrics-server/README.md

```diff
@@ -29,6 +29,7 @@ Parameter | Description | Default
 `priorityClassName` | Pod priority class | `""`
 `readinessProbe` | Container readiness probe | See values.yaml
 `service.annotations` | Annotations to add to the service | `{}`
+`service.labels` | labels to be added to the metrics-server service | `{}`
```

Comment:
```suggestion
`service.labels` | Labels to be added to the metrics-server service | `{}`
```

Reply:
I recommend applying this manually and force pushing this :).

Reply:
why edit this only, as opposed to all of these?

Reply:
I think you misunderstood my review comment. The uppercasing of the first letter was only meant for the Description, not the value field.

Reply:
Yep, I sure did.